tools/functional-tester/etcd-tester/stresser.go

```diff
@@ -71,49 +71,63 @@ func (st *stressTable) choose() stressFunc {
 	return st.entries[idx].f
 }
 
-func newStressPut(kvc pb.KVClient, keySuffixRange, keySize int) stressFunc {
+func (s *stresser) newStressPut(kvc pb.KVClient) stressFunc {
```

Comment:
There's no need to tie these to a specific stresser. Once the stressFunc is created, the input values are sealed. Otherwise, it implies that the stresser will occasionally update `s.keySuffixRange` after creating the stressFunc, which isn't the case.